### PR TITLE
Add Short collect cell

### PR DESCRIPTION
**Pull request by EliyaFishman**

### CODE REVIEW

File: src/main/java/frc/robot/commands/command_groups/ShortCollectCell.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+public class ShortCollectCell extends SequentialCommandGroup {
+
+    private static final int kCollectTime = 2;
+
+    public ShortCollectCell() {
+        addCommands(
+            new StartEndCommand(() -> led.setColor(LEDColor.Orange), () -> led.turnOffLED(), led),
+            parallel(
+                new SetIntakeState(true),
+                new SetIntakeSpeed(robotConstants.intakeConstants.kDefaultIntakePower)
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/commands/command_groups/ShortCollectCell.java]: not finished, also - new SetIntakeSpeed(robotConstants.intakeConstants.kDefaultIntakePower)
use regular constructor

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
do fix

**Comment by EliyaFishman** [comment]
It now has a new enum IntakeState that describes the desired angle of the intake opener

**Comment by EliyaFishman** [comment]
[Change open intake to work with enum - IntakeAngle](https://trello.com/c/UJ14JFa9/168-change-open-intake-to-work-with-enum-intakeangle)

**Review by EliyaFishman — COMMENTED** [review]
Good

**Review by HilelAmiel — APPROVED** [review]

**Pull request merged**